azure-pipelines.yml

```diff
@@ -31,16 +31,20 @@ steps:
 - script: |
     python -m pip install --upgrade pip wheel twine setuptools_scm
     pip install numpy
-  displayName: 'Install Dependencies'
+  displayName: 'Install dependencies'
+
+- script: |
+    git clone https://github.com/xdf-modules/example-files.git
```

Comment:
This should be a shallow clone, or simply fetch https://github.com/xdf-modules/example-files/archive/master.zip, we don't need the entire history.

Reply:
Done, changed to shallow clone.